docs/Version release notes.md

```diff
@@ -7,6 +7,28 @@ pagination_prev: null
 Version Release Notes
 =====================
 
+Release date Jul 1, 2024
+-------------------------
+
+### Android Point SDK 16.1.0
+
+This update includes several bug fixes and improvements to enhance the functionality and stability of the SDK.
+
+#### What's new
+* **Deprecated GeoTriggerService.isRunning() API:** 
+    * The existing `GeoTriggerService.isRunning()` API has been deprecated.
+    * A new API `GeoTriggerService.isRunning(context: Context)` has been introduced for improved context handling.
+* **GeotriggerEvent Class Update:**
```

Comment:
Exact class name is `GeoTriggerEvent`. Please update it.

Reply:
Updated!